### PR TITLE
Update b3dm format and add new deprecated tilesets

### PR DESCRIPTION
**Pull request by lilleyse**
For https://github.com/AnalyticalGraphicsInc/cesium/pull/5165

### CODE REVIEW

**Comment by pjcozzi** [comment]
@shehzan10 can you please review and merge when you are available?

**Comment by shehzan10** [comment]
I'm not entirely convinced that using `deprecated1` and `deprecated2` is a good. idea. I don't think there is enough intuitive distinction between then two.

One possible was to rename these are `options.header20Bytes` and `options.header24Bytes`. Assuming there is no versioning to the deprecated spec, I think we should use something more intuitive.

But otherwise looks good to me.

**Comment by lilleyse** [comment]
Added some updates here.

* I realized the b3dm padding was not completely correct and all the glb's will start on a four byte boundary instead of eight. Doesn't really matter since the glbs here don't require it. Also not worth opening another data PR into Cesium for just that.
* Fixed issue where the `createBatchTableBinary` flag was being ignored for i3dm. Affects just one tileset, also not a huge deal.

> I'm not entirely convinced that using deprecated1 and deprecated2 is a good. idea. I don't think there is enough intuitive distinction between then two.

I think this is ok for now. This does represent the versions over time, so its easy for me to remember at least and isn't too wordy.

File: generator/lib/createB3dm.js [hunk 1]
```diff
@@ -13,43 +13,63 @@ module.exports = createB3dm;
  * @param {Object} options An object with the following properties:
  * @param {Buffer} options.glb The binary glTF buffer.
  * @param {Number} [options.batchLength] The number of features in the tile.
```
Comment (shehzan10) [inline generator/lib/createB3dm.js]:
```
options.batchLength
```
This option should be removed, right?

Comment (lilleyse) [inline generator/lib/createB3dm.js]: Yup, removed.
